src/app/gateway/gateway-table/gateway-table.component.html

```diff
@@ -1,98 +1,133 @@
+<app-column-selector
+  [(displayedColumns)]="displayedColumns"
+  [columnDefinitions]="columnDefinitions"
+  [localStorageKey]="gatewayTableSavedColumns"
+></app-column-selector>
+
 <div class="mat-elevation-z8">
-    <div class="loading-shade" *ngIf="isLoadingResults">
-        <mat-spinner *ngIf="isLoadingResults"></mat-spinner>
-    </div>
-    <table mat-table [dataSource]="dataSource" matSort matSortActive="name" matSortDirection="asc" matSortDisableClear>
+  <div class="loading-shade" *ngIf="isLoadingResults">
+    <mat-spinner *ngIf="isLoadingResults"></mat-spinner>
+  </div>
+  <table mat-table [dataSource]="dataSource" matSort matSortActive="name" matSortDirection="asc" matSortDisableClear>
+
+    <!-- Name Column -->
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>
+        {{ 'LORA-GATEWAY-TABLE.NAME' | translate }}
+      </th>
+      <td mat-cell *matCellDef="let gateway">
+        <a [routerLink]="'/gateways/gateway-detail/' + gateway.gatewayId" routerLinkActive="active" class="application-link">
+          {{ gateway.name }}
+        </a>
+      </td>
+    </ng-container>
+
+    <!-- Id Column -->
+    <ng-container matColumnDef="gateway-id">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'LORA-GATEWAY-TABLE.GATEWAYID' | translate }}</th>
+      <td mat-cell *matCellDef="let gateway">
+        {{ gateway.gatewayId }}
+      </td>
+    </ng-container>
+
+    <!-- Organization Column -->
+    <ng-container matColumnDef="internalOrganizationName">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'LORA-GATEWAY-TABLE.ORGANIZATION' | translate }}
+      </th>
+      <td mat-cell *matCellDef="let gateway">
+        {{ gateway.organizationName }}
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="rxPacketsReceived">
+      <th mat-header-cell *matHeaderCellDef>{{ 'LORA-GATEWAY-TABLE.PACKETS-RECEIVED' | translate }}</th>
```

Comment:
In my opinion, it should be more clear that packets received and send is only from today :-).

Reply:
We can discuss this with Mogens